documentation/docs/migration-guide.md

```diff
@@ -74,20 +77,67 @@ Access to node information such as name, bounding box and properties/attributes
 
-## Other differences
+### Other differences
 
 There are a few other noticeable changes from `@cognite/3d-viewer` and `@cognite/reveal`:
 
 - `@cognite/3d-viewer` supports local caching to reduce the time to load previously opened 3D models. Currently, this is not supported by `@cognite/reveal`, but the need for such functionality is reduced by adding streaming capabilities.
 - In `@cognite/3d-viewer` `Cognite3DViewer.addModel(...)` will always return a `Cognite3DModel`. In `@cognite/reveal` this function might also return a `CognitePointCloudModel`. To explicitly add a CAD model or point cloud model use `Cognite3DViewer.addCadModel(...)` or `Cognite3DViewer.addPointCloudModel(...)`
-- `Cognite3DViewer.loadCameraFromModel(...)`] has been added for loading camera settings from CDF when available.
\ No newline at end of file
+- `Cognite3DViewer.loadCameraFromModel(...)`] has been added for loading camera settings from CDF when available.
+
+## Preparing models
+
+`@cognite/reveal` requires new output file formats which means that 3D models that have been converted before June 2020 might need reconversion. To determine if a model needs reconversion, the [listModelRevisionOutputs API endpoint](...) can be used with URL parameter `format=reveal-directory`. This can be done using the SDK, e.g.:
+```jsx
+// Switch these with your project/model identifier
+const project = 'publicdata';
+const modelId = 4715379429968321;
+const revisionId = 5688854005909501; 
+
+// Prepare a SDK client and authenticate
+const sdk = new CogniteClient({ appId: 'cognite.reveal.docs.IsModelCompatible' });
+sdk.loginWithOAuth({ project });
+await sdk.authenticate();
+
+// Request a list of formats
+const url = `https://api.cognitedata.com/api/v1/projects/${project}/3d/models/${modelId}/revisions/${revisionId}/outputs`;
+const response = await sdk.get(url, { params: { format: 'reveal-directory' } });
+
+// Determine if there's compatible outputs
+const hasCompatibleOutputs = response.data.items.length > 0;
+alert(`Is model compatible with @cognite/reveal: ${hasCompatibleOutputs}`);
+```
+
+Or by using `curl` (requires a [Cognite API key or access token](https://docs.cognite.com/dev/guides/iam/authentication.html#api-keys)):
+
+```bash
+> curl "https://api.cognitedata.com/api/v1/projects/$PROJECT/3d/models/$MODELID/revisions/$REVISIONID/outputs?format=reveal-directory" --header 'api-key: $APIKEY'
+
+{
+  "items": [
+    {
+      "format": "reveal-directory",
+      "version": 8,
+      "blobId": 42723775736403107
+    }
+  ]
+}
+```
+where `PROJECT`, `MODELID`, `REVISIONID` and `APIKEY` must be set in the environment variables. If the model isn't compatible the response will be an empty list, i.e.
+```bash
+{
+  "items": []
+}
+```
+
+If the model isn't compatible with `@cognite/reveal`, it must be reprocessed. This can be done by [uploading a new revision in Cognite Fusion](https://docs.cognite.com/cdf/3d/guides/3dmodels_upload.html).
```

Comment:
```suggestion
If the model isn't compatible with `@cognite/reveal`, it must be reprocessed. This can be done by [uploading a new revision in Cognite Data Fusion](https://docs.cognite.com/cdf/3d/guides/3dmodels_upload.html).
```

see https://cognitedata.slack.com/archives/C3Y6LU1QV/p1599649299114800